### ISSUE TITLE
A request has failed. See the output for more information.

### ISSUE THREAD

**Issue by Lnncoco** [denoland/vscode_deno#492]
**Describe the bug**
deno v1.12.2   extension v3.8.0

![image](https://user-images.githubusercontent.com/57341815/128965229-c0b64ba4-4c70-41e7-b0bd-05534ae25585.png)
A request has failed. See the output for more information.

**To Reproduce**
任意ts文件,输入中文,或者中文英文切换着输入的时候, 出现的比较随机 但印象往往都有带中文或者中文符号的时候

[Error - 上午11:22:44] Request textDocument/codeLens failed.
  Message: Internal error
  Code: -32603 
{"type":"mark","name":"code_lens","count":2167,"args":{"textDocument":{"uri":"file:///d%3A/Project/deno-test/modules/semver.ts"}}},
{"type":"mark","name":"get_line_index_sync","count":5993,"args":{"specifier":"file:///D:/Project/deno-test/modules/semver.ts"}},
{"type":"measure","name":"get_line_index_sync","count":5993,"duration":0},
{"type":"mark","name":"get_navigation_tree","count":2167,"args":{"specifier":"file:///D:/Project/deno-test/modules/semver.ts"}},
Error getting code lenses for "file:///D:/Project/deno-test/modules/semver.ts": failed to send request to tsc thread
[Error - 上午11:22:44] Request textDocument/codeLens failed.
  Message: Internal error
  Code: -32603

**Comment by kitsonk** [comment]
There are some known issues with editing documents with high surrogate pairs, as the text diffing library has an issue with it (https://github.com/dtolnay/dissimilar/issues/5).

Would it be possible to get the text of a file just as the error occurs?  I assume it occurs at you are typing in the document?

**Comment by Lnncoco** [comment]
yes, Input mixed Chinese characters  and English characters . Non-inevitable occurrence

**Comment by bartlomieju** [comment]
Is this still reproducible with latest Deno version?
